From aed87d1ffe2d22986e38b22b1c77beadcd3376c8 Mon Sep 17 00:00:00 2001
From: Jae Hyun Yoo <jae.hyun.yoo@intel.com>
Date: Fri, 30 Jul 2021 14:17:24 -0700
Subject: [PATCH] ARM: dts: add rtc-pch node into aspeed-bmc-intel-ast2xxx

This commit adds rtc-pch node into aspeed-bmc-intel-ast2500 and
aspeed-bmc-intel-ast2600 to enable PCH RTC driver.

Signed-off-by: Jae Hyun Yoo <jae.hyun.yoo@intel.com>
---
 arch/arm/boot/dts/aspeed-bmc-intel-ast2500.dts | 5 +++++
 arch/arm/boot/dts/aspeed-bmc-intel-ast2600.dts | 5 +++++
 2 files changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed-bmc-intel-ast2500.dts b/arch/arm/boot/dts/aspeed-bmc-intel-ast2500.dts
index 2efa67f08dcb..a460169e1b7b 100644
--- a/arch/arm/boot/dts/aspeed-bmc-intel-ast2500.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-intel-ast2500.dts
@@ -383,6 +383,11 @@
 	aspeed,dma-buf-size = <4095>;
 	aspeed,hw-timeout-ms = <300>;
 	status = "okay";
+
+	rtc-pch@44 {
+		compatible = "rtc,pchc620";
+		reg = <0x44>;
+	};
 };
 
 &i2c4 {
diff --git a/arch/arm/boot/dts/aspeed-bmc-intel-ast2600.dts b/arch/arm/boot/dts/aspeed-bmc-intel-ast2600.dts
index 02a671ca0f19..9bc859f69307 100644
--- a/arch/arm/boot/dts/aspeed-bmc-intel-ast2600.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-intel-ast2600.dts
@@ -491,6 +491,11 @@
 	multi-master;
 	aspeed,hw-timeout-ms = <300>;
 	status = "okay";
+
+	rtc-pch@44 {
+		compatible = "rtc,pchc620";
+		reg = <0x44>;
+	};
 };
 
 &i2c12 {
-- 
2.17.1

